skimage/restoration/_denoise.py

```diff
@@ -286,8 +290,25 @@ def denoise_tv_bregman(image, weight, max_iter=100, eps=1e-3, isotropic=True):
     shape_ext = (rows + 2, cols + 2, dims)
 
     out = np.zeros(shape_ext, image.dtype)
-    _denoise_tv_bregman(image, image.dtype.type(weight), max_iter, eps,
-                        isotropic, out)
+    channel_out = np.zeros(shape_ext[:2] + (1,), dtype=out.dtype)
```

Comment:
This should only be allocated if multichannel is true

Reply:
```suggestion
```